Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -550,6 +550,8 @@ class RenderStyle {
 
     ContentVisibility contentVisibility() const { return static_cast<ContentVisibility>(m_rareNonInheritedData->contentVisibility); }
 
+    bool isInSkippedContentSubtree() const { return m_rareInheritedData->isInSkippedContentSubtree; }
```

Comment:
Something like effectiveSkipsContent might be less clunky and communicate that this is an effective property (similar to others).

Reply:
Done.